readthedocs/builds/migrations/0003_add-cold-storage.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-10-04 17:27
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import readthedocs.builds.version_slug
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('builds', '0002_build_command_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='build',
+            name='cold_storage',
+            field=models.BooleanField(default=False, help_text='Build comamnds are stored outside the database.', verbose_name='Cold Storage'),
+        ),
+        migrations.AlterField(
```

Comment:
All this was added automatically, I think small updates that we haven't migrated on.

Reply:
Yeah, i get some of these too. Lets remove them from here and figure out which of these migrations are required now. We might need to roll a fresh db locally to figure that out.